specification/nginx/resource-manager/NGINX.NGINXPLUS/stable/2023-09-01/swagger.json

```diff
@@ -0,0 +1,1579 @@
+          "$ref": "#/definitions/NginxDeploymentUpdateProperties"
+        }
+      }
+    },
+    "NginxDeploymentListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NginxDeployment"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "The object that represents the operation.",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Service provider: Nginx.NginxPlus",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Type on which the operation is performed, e.g., 'deployments'.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Operation type, e.g., read, write, delete, etc.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the operation, e.g., 'Write deployments'.",
+          "type": "string"
+        }
+      }
+    },
+    "OperationResult": {
+      "description": "A Nginx.NginxPlus REST API operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of GET request to list Nginx.NginxPlus operations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of operations supported by the Nginx.NginxPlus provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResult"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string"
+        }
+      }
+    },
+    "AnalysisCreate": {
+      "type": "object",
+      "description": "The request body for creating an analysis for an NGINX configuration.",
+      "properties": {
+        "config": {
+          "type": "object",
+          "properties": {
+            "rootFile": {
+              "type": "string",
+              "description": "The root file of the NGINX config file(s). It must match one of the files' filepath."
+            },
+            "files": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NginxConfigurationFile"
+              },
+              "x-ms-identifiers": []
```

Comment:
>"x-ms-identifiers" [](http://example.com/codeflow?start=14&length=18)

You can identify the key order of the collection by adding [for example] virtualPath here.